### PR TITLE
Update NeuroFly_Toolkit.md

### PR DESCRIPTION
**Pull request by jartuso**

### CODE REVIEW

File: website/docs/Software/OpenBCISoftware/NeuroFly_Toolkit.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ This tutorial will show you how to control a joystick using EMG data with the Op
 3.  [EMG/ECG Snap Electrode Cables](https://shop.openbci.com/collections/frontpage/products/emg-ecg-snap-electrode-cables?variant=32372786958)
 4.  Computer with downloaded [OpenBCI GUI](Software/OpenBCISoftware/01-OpenBCI_GUI.md). **Be sure to use [OpenBCI GUI v5.2.0](https://github.com/OpenBCI/OpenBCI_GUI/releases/latest) or later**.
 
+## Video Explainer
```
Comment (retiutut) [inline website/docs/Software/OpenBCISoftware/NeuroFly_Toolkit.md:15]: Can we rename this heading?

**Review by retiutut — CHANGES_REQUESTED** [review]
Request changing a header title.

File: website/docs/Software/OpenBCISoftware/NeuroFly_Toolkit.md (hunk 1 above)
Comment (jartuso) [reply website/docs/Software/OpenBCISoftware/NeuroFly_Toolkit.md]: Yes. I changed it in a new commit, but feel free to make it something else

**Review by retiutut — APPROVED** [review]

**Comment by retiutut** [comment]
Thanks Joe! Approved!

**Pull request merged**